Sources/TIM/AppAuth/AppAuthController.swift

```diff
@@ -243,5 +238,34 @@ public final class AppAuthController: OpenIDConnectController {
             completion(result)
         }
     }
+    
+    /// Create AuthorizationRequest using the authorizationRequestNonce in case it is provided
+    private func createAuthorizationRequest(config: OIDServiceConfiguration, authorizationRequestNonce: String?) -> OIDAuthorizationRequest {
+        if authorizationRequestNonce == nil {
+            return OIDAuthorizationRequest(
+                configuration: config,
+                clientId: self.credentials.clientId,
+                scopes: self.credentials.scopes,
+                redirectURL: self.credentials.redirectUri,
+                responseType: OIDResponseTypeCode,
+                additionalParameters: [:]
+            )
+        } else {
+            return OIDAuthorizationRequest(
+                configuration: config,
+                clientId: self.credentials.clientId,
+                clientSecret: nil,
+                scope: self.credentials.scopes.joined(separator: " "),
```

Comment:
We should use `OIDScopeUtilities.scopes(with: self.credentials.scopes)` here. 

https://github.com/openid/AppAuth-iOS/blob/98ed5e15ceedcacc0b3fd6c3521a22411acc2212/Source/AppAuthCore/OIDAuthorizationRequest.m#L178